<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BidDialog</class>
 <widget class="QDialog" name="BidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bid Dialog</string>
  </property>
  <widget class="ScaledQLabel" name="player1Label" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string></string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player3Label" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string></string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player2Label" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string></string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player4Label" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string></string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player2BidLabel" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>0</string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player3BidLabel" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>0</string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player4BidLabel" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>0</string>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="player1BidLabel" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>0</string>
   </property>
  </widget>
  <widget class="ScaledQPushButton" name="bidButton" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>Bid</string>
   </property>
  </widget>
  <widget class="ScaledQPushButton" name="passButton" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>Pass</string>
   </property>
  </widget>
  <widget class="ScaledQComboBox" name="bidAmountComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="ScaledQLabel" name="bidAmountLabel" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>Bid Amount</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledQLabel</class>
   <extends>QLabel</extends>
   <header>scaledWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledQPushButton</class>
   <extends>QPushButton</extends>
   <header>scaledWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledQComboBox</class>
   <extends>QComboBox</extends>
   <header>scaledWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
